docs/src/pages/index.mdx

```diff
@@ -0,0 +1,178 @@
+---
+layout: ../layouts/wiki.astro
+---
+
+import InstallCommands from "@/components/install-commands.astro";
+
+# Remeda
+
+Fully written in TypeScript for powerful, type-safe, and functional utilities.
```

Comment:
**suggestion:** maybe something like

```suggestion
Powerful, type-safe, functional utilities for TypeScript.
```

Reply:
maybe we can even make it bigger font-size-wise or something, but can think about that later

Reply:
My issue with "for TypeScript" is that it doesn't differentiate between runtime and typing, type-fest and ts-toolbelt are TypeScript utility libraries, they focus on something different than us, and we don't offer exported types at all.

Reply:
but "Fully written in TypeScript" doesn't do that either, i think. the main reason i made this suggestion was that it feels weird to lead the tagline with that, the fact that our library is "fully written" in typescript isn't as important as, say, the fact that it's type-safe, or typescript-first, or whatever

i also think "functional utilities" already says that it's a runtime level utility library. compare [ts-belt's tagline](https://github.com/mobily/ts-belt) ("Fast, modern, and practical utility library for FP in TypeScript.") or [radash's tagline](https://github.com/sodiray/radash) ("Functional utility library - modern, simple, typed, powerful")

we could put "for TS/JS" like, say, [tanstack query's tagline](https://tanstack.com/query/latest) ("Powerful asynchronous state management for TS/JS, React, Solid, Vue, Svelte and Angular")

Reply:
I didn't like the tagline either, but after trying to come up with something for a while, this felt like the one that covered my main points most closely. Let's try to come up with a better one; I just couldn't.

I don't want TS/JS because we don't want to support non-TS usage. Without typing, you can run into issues where we don't do runtime validation on inputs, and the code could explode. I recommend that people on JS projects not use Remeda and prefer Lodash, which does runtime validation (at least sometimes).

I mentioned that the library is written in TypeScript because none of the big libraries are. I thought this would communicate to people who think that a TypeScript codebase (like their own) is a higher-quality project than one in JS, so a library that is fully written in TypeScript is a sign of its quality.

Reply:
alright then! this wasn't a blocking comment anyway